third_party/move/move-compiler-v2/Cargo.toml

```diff
@@ -10,6 +10,7 @@ publish = false
 edition = "2021"
 
 [dependencies]
+abstract-domain-derive = { path = "../move-model/bytecode/abstract_domain_derive" }
```

Comment:
I'm confused. I reviewed another PR with this stuff, namely https://github.com/aptos-labs/aptos-core/pull/11674. Are you wanted to compare against this PR?

Reply:
done